client/components/settings/AccessTokenModal.vue

```diff
@@ -0,0 +1,141 @@
+<template>
```

Comment:
When we create a token, the save button should not be there anymore (eventually replace with close button)
![Replace save with](https://github.com/user-attachments/assets/206a7268-b61e-42a6-9232-837cb8f16296)

Let's replace the success message with a [Nuxt UI alert (subtle green)](https://ui.nuxt.com/components/alert#style)
![CleanShot 2024-07-28 at 20 16 45@2x](https://github.com/user-attachments/assets/caf4cd41-1e92-4022-bdf1-ec1031444ee2)